README.md

```diff
@@ -84,6 +138,8 @@ Add assets, e.g. images or JavaScript, to [`assets`](assets).
 [`favicon.ico`](favicon.ico) should be served from the root of the project.
 Just replace the current one with the `favicon.ico` of your project.
 
+
+
 ## License
```

Comment:
I believe @JustinCappos proposed changing the license. We should probably have a discussion regarding the licensing overall. For now we can keep text as is and fix in a pull request fixing the dedicated issue.